securedrop_client/gui/state.py

```diff
@@ -0,0 +1,49 @@
+import textwrap
+from typing import Dict, List, NewType, Optional
+from uuid import UUID
+
+from PyQt5.QtCore import QObject
+
+ConversationId = NewType("ConversationId", UUID)
+FileId = NewType("FileId", UUID)
```

Comment:
we need to verify the compatibility of UUID and each `NewType` we create against each corresponding db type, e.g. for `File` uuid as defined in `db.yp`: `Column(String(36), unique=True, nullable=False)`

Reply:
I'm 95% sure **mypy** is checking for that when we cast one into the other.